### DISCUSSION TITLE
European PCB/Case group buys

### DISCUSSION THREAD

**Discussion by pixiesue** [FundamentalFrequency/.github#6] [General]
As said the Gerber files are available but i don't know (yet) what specs to order or even which manufacturer to use. 

Thank you to @FundamentalFrequency for sharing this exciting project!

Do the Gerber files alone enough for the manufacturer to do the job?
Do I need to provide any other specs? Any advice is welcome!!
Here are a few of what seems to be important :

Nb of Layers = 2
PCB Thickness = 1.5mm (?)
TG = 170° (?)
External Copper Thickness = 17/35μm

PCB Length/Width
Surface Treatment
Solder Mask

**Comment by stonepreston** [comment]
I used JLCPCB when I ordered my first boards. The zip archive contained in the ECAD repository releases contains both the Gerber files and the drill files. You should be able to upload the zip archive to whatever board manufacturer you choose and that should be all you need.

**Comment by pixiesue** [comment]
Hello,
So, I ran the gerbers.zip in the JLCPCB configurator and well... The quote is cheap, so cheap I suspect I have forget something.
I am trying to find local manufacturers but for the moment their prices are insanely high.
I guess I will give JLCPCB a go.

![image](https://user-images.githubusercontent.com/107801377/174574218-0bab04b8-758c-46f1-8017-31d2329bf967.png)

**Reply by stonepreston** [reply]
That seems reasonable to me.

**Comment by justinwoods2k** [comment]
Thank you for looking into this, @pixiesue. And thank you @stonepreston for everything you have done on this incredible project so far. Please count me in for a European group buy on the PCB only. I’m in Berkshire, UK and looking to make the enclosure by hand.

**Reply by pixiesue** [reply]
I would love a hand made enclosure, please share your design!

**Comment by ChappIO** [comment]
Yes, I'd be willing to join in (from NL)

**Comment by winfr34k** [comment]
Yep, would like to join (Germany)... Would need both.

**Comment by scunner3rd** [comment]
I'd be up for one too (UK)

**Comment by jakemorgangit** [comment]
Same here for the European PCB group buy (UK based)

**Comment by Apfelwurm** [comment]
would also be interested in a pcb as well as the case parts if we manage to get both. Located in Germany :)

**Comment by schurig-fabian** [comment]
I am also interested in the pcb. I am not really interested in the case parts, as I would try to contribute a 3D printable case. I am located in Germany

**Reply by pixiesue** [reply]
please share your design! There is interest for that among us.

**Comment by jmorzeck** [comment]
I am interested in both pcb and case parts, located in Germany!

**Reply by jderiz** [reply]
same

**Comment by mroemore** [comment]
I too would like to join.

**Comment by schegges** [comment]
I am also interested in two PCB‘s only. located in Germany as well.

**Comment by maxupp** [comment]
Interested in 3 each of PCB and case parts. (Germany)

**Comment by PaulREE** [comment]
Hi!

I'm from Ukraine. I work for a company that does PCB design, PCB assembly, BOM ordering and CNC machining services. So feel free to contact me.

**Reply by lstep** [reply]
Would your company be able to create the PCB, case parts, and manage the BOM? For what price?

**Reply by PaulREE** [reply]
Hi @lstep 

Yes, this is our speciality. To find out the cost of your project, please send your project details to us. We help startups or other clients create mechanical and electronic prototypes.

<details>
  <summary>About Us</summary>
  
Alnicko Lab is a full cycle hardware development company. We have clients all over the world, for example from the USA, Canada, Germany, Israel and Australia.
Our research and development center is located in Ukraine. There are work 50 engineers. We have embedded development team, schematics and PCB design team, industrial and mechanical team. All other stuff: project managers, production managers, logistic managers, etc.

Main technologies: 
– Embedded design: embedded C development for different micro controllers (like STM32, Texas Instruments, etc.), Python development (Linux/Android platforms), porting OS Linux, Android to customer’s platforms, OS optimization for the needed functionality and energy consumption, drivers development for external interfaces (audio, video, SPI/I2C/CAN/UART sensors, etc.);
– Schematics & PCB Design: wireless (Wi-Fi, Bluetooth, LoraWAN, NFC, NB-IoT,  GSM/GPRS/EDGE, ZigBee, MiWi, 3G, LTE, infrared, etc.), customized motherboards on multi-core processors (rigid and flexible PCB, multilayer design), different processors (in most cases ARM architecture), digital interfaces (DDR3, MIPI, SDIO, SPI, I2S, USB, UART, I2C, RS232, ModBus, RS485/RS422, CAN, CANOpen, etc.), video- and audio-processing, power supply (Li-ion/Li-pol batteries, AC/DC/ DC/DC, wireless charging, etc.);
– Industrial and mechanical design: 3D Studio & Solidworks, prototyping on 3D printers and casting into temporary forms, choosing plastics, ceramics, metals in accordance with operation conditions (temperature, humidity, underwater usage), pre-production and help in manufacturing process.

Today we have more than 30 projects of various levels of complexity, many of which include disruptive technologies. Like DataScience in video, or ultrasound, spectrometry, etc. However, they are all based on microcontrollers or processors (CPU). There are digital electronics. Niche areas: healthcare, industrial, agroindustrial (local GPS, sensors), consumer products, wearables, etc. Success in achieving project goals is quite high (95-97%).

Our strengths are 3 key technologies:
– video- and audio-processing
– wireless technologies (all popular techs)
– Li-ion, Li-Pol power sources (for wearables).

Visit our website: [alnicko.com](http://alnicko.com/)
  
</details>

As for open source projects that interested me, I can help you organize the production of parts on a 3D printer (HP MJF, FDM, SLA) or CNC machine in Ukraine at a good price.

**Reply by Apfelwurm** [reply]
This sounds super nice! What details do you need to know besides the stuff thats already made public by FundamentalFrequency to determinate the costs? :)

**Comment by nelarsen97** [comment]
I would be interested in getting a PCB and case parts as well. I'm in Norway

**Comment by lstep** [comment]
Interested in PCB and case parts also. I'm from France

**Comment by tandy-1000** [comment]
Interested in all parts! I'm from the UK.

**Comment by reklm-e** [comment]
Could i also join?

**Comment by dun198** [comment]
Interested as well in the PCB -> GER

**Comment by pixelscript** [comment]
PCB + parts here too (UK)

**Comment by juliandoerner** [comment]
Hey, I would be in for a PCB. I'm from Germany.

**Comment by Terrz** [comment]
Interested as well(uk)

**Comment by Ehoax** [comment]
Interested in a PCB or 2 (Berlin, Germany)

**Comment by fofinho666** [comment]
I’m also interested! I’m from Portugal

**Comment by TW8** [comment]
I would also be interested in an european group buy for the PCB and possibly also the case! (Germany)

**Comment by jmorzeck** [comment]
Since there have been many people replying to this thread, what would be the number of orders to get this started? 
@pixiesue Did you already order from JLCPCB?

**Comment by mackemint** [comment]
Swede here interested in a group buy! Love synth DIY and this project looks very fun :)

**Comment by pixiesue** [comment]
Hello guys,

Thank you all for your interest in @stonepreston project.
We are not affiliated with him, like you, we think it is more 
than just a neat project and we want to contribute as we can.

We use an unofficial discord to iron out the details of the european 
group buys, we also try to help each others with all the how to.

We do it on discord to not "pollute" github with nonsense, feel free to join us there but we'll try to keep that forum up to date.

https://discord.gg/7ZjSW84C

We are a few people to have already ordered from JPCLB. ETA : soon™ :)

10 white from France
50 blue boards from UK
10 black from Germany
maybe yours...
We have enough for Europe at the time (someone suggested yellow for sweden)

UK
@justinwoods2k 
@scunner3rd
@jakemorgangit
@tandy-1000
@pixelscript
@Terrz

GERMANY
@winfr34k 
@Apfelwrum
@Mayjayday
@kastenfrosch
@jderiz
@schegges x2
@maxupp x3
@dun198
@juliandoerner
@Ehoax x2
@TW8
@paulnato

FRANCE
@pixiesue 
@Istep

NORWAY
@nelarsen97

SWEDEN
@mackemint

PORTUGAL
@fofinho666

UNKNOWN
@mroemore
@reklm-e

**Reply by alexeynefdef** [reply]
Hi! This link has expired

**Comment by louisnichols** [comment]
+1 for Germany. Is joining Discord the only way? I haven't used it before.

**Reply by schegges** [reply]
Ich hab grad welche bestellt und könnte dir eins davon reservieren. Wie kann ich dich denn kontaktieren für Updates und wegen Versand sobald die Boards hier sind?

**Reply by pixiesue** [reply]
Hello,
The discord is non official, but it is an easier, more casual way to exchange about it, and to maybe get help to build your LMN later.
Feel free to join us. Anyway there's enough boards ordered for the actual european group buy, you'll find one of your choice color easily ;)

**Comment by MR-B0NES** [comment]
Hey all,

I'm selling PCBs from my order to UK peeps here.

https://synthsoup.com/products/lmn-3-pcb?utm_source=copyToPasteBoard&utm_medium=product-links&utm_content=web

@justinwoods2k
@scunner3rd
@jakemorgangit
@tandy-1000
@pixelscript
@Terrz

**Reply by justinwoods2k** [reply]
Thank you, @MR-B0NES. I had some ill health that prevented me from being more proactive on this so I really appreciate you arranging and distributing PCBs in the UK. Order placed.

**Reply by StalkS-24x7** [reply]
Hi looking to pick up a pcb in the UK - do you have any left from your order?

**Comment by maxupp** [comment]
I have ordered 30 PCBs , around 10 of which are currently unspoken for. Will ship within EU for around 4€ each + 6€ shipping.

**Reply by schurig-fabian** [reply]
I am interested in one. How can I send you money and details?

**Reply by maxupp** [reply]
Easiest way is on discord, you'll find instruction in the "Europe" group buy channel.

**Reply by farcaller** [reply]
the discord link seem to have expired.

**Reply by maxupp** [reply]
https://discord.gg/73dqDEdu

**Reply by mrWheel** [reply]
I am also interested in a pcb as well as the case parts. Located in The Netherlands

**Comment by mrWheel** [comment]
I am also interested in a pcb as well as the case parts. Located in The Netherlands

**Reply by theturboferret** [reply]
I have 4 spare boards if you are still interested

**Reply by louisnichols** [reply]
@theturboferret I'm also interested. I'll email you.

**Reply by charlydev42** [reply]
Hey, do you by any chance still have some of these pcbs ?

**Comment by alexeynefdef** [comment]
Hi! Definitely interested!!

On Tue 23. May 2023 at 15:59, theturboferret ***@***.***>
wrote:

> I have 4 spare boards if you are still interested
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/FundamentalFrequency/discussions/6#discussioncomment-5978784>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APFGOHAWFNZWCJKRK3ESOZTXHS7CNANCNFSM5ZGQFL3Q>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>
-- 

Best regards

Alexey Nefedov

instagram x ick.bin <https://www.instagram.com/ick.bin/>

**Reply by theturboferret** [reply]
perfect, send me an email turboferret@gmail.com

**Comment by theturboferret** [comment]
I don't have any boards left, but do have a completed unit available.

Let me know if that would be of interest.

Cheers, Richard

**Reply by StalkS-24x7** [reply]
Hey Richard - Potentially - What's the best way to get hold of you (email, discord, etc.)?

**Comment by theturboferret** [comment]
email me ***@***.***

Cheers, Richard

**Comment by theturboferret** [comment]
I have a complete unit available if you are interested, but no pcbs I'm afraid.  Cheers, Richard
________________________________

**Comment by panki27** [comment]
Hi, also looking for a PCB in Europe. 
@theturboferret is your completed unit still available?

**Comment by theturboferret** [comment]
Yes,My complete unit is still available  Cheers, Richard Sent from my iPhoneOn 28 Jul 2025, at 17:24, Felix Pankratz ***@***.***> wrote:﻿
Hi, also looking for a PCB in Europe.
@theturboferret is your completed unit still available?
